config/proto_conversions.go

```diff
@@ -106,6 +106,10 @@ func FromProto(proto *pb.RobotConfig, logger logging.Logger) (*Config, error) {
 		cfg.MaintenanceConfig = maintenanceConfig
 	}
 
+	if proto.DisableLogDeduplication {
```

Comment:
I agree this is correct because `cfg` is default initialized, but this feels like the seedling of a bug. `cfg.DisableLogDeduplication = proto.DisableLogDeduplication` is less lines and isn't prone to the default initialization value assumption.

Reply:
...I have no idea what I was thinking here haha. Switched to `cfg.DisableLogDeduplication = proto.DisableLogDeduplication`.